javascript/002-objetos-js-em-detalhe.md

```diff
@@ -3,91 +3,91 @@
 * **Artigo original**: [JavaScript Objects in Detail](http://javascriptissexy.com/javascript-objects-in-detail/)
 * **Tradução**: [Eric Douglas](https://github.com/ericdouglas)
 
-O núcleo mais utilizado e mais básico do JavaScript é o Object. O JavaScript tem um tipo complexo de dado, o tipo Object, e tem cinco tipos de dados simples: Number, String, Boolean, Undefined e Null. Note que estes tipos simples (primitivos) de dados são imutáveis, eles não podem ser alterados, enquanto os objetos podem.
+O tipo de dado central, o mais fundamental e freqüentemente utilizado, do JavaScript é o tipo Object. O JavaScript tem 1 tipo de dado complexo, o tipo Object, e tem 6 tipos de dados simples: Boolean, Number, String, Symbol, undefined e null. Note que estes tipos de dados simples (primitivos) são imutáveis (não podem ser alterados), enquanto os objetos são mutáveis (podem ser alterados).
```

Comment:
Ainda acho que no caso de `Symbol`, deveria haver ao menos uma nota de rodapé mencionando o seu acréscimo no ECMA 6.
Informação defasada é bastante inútil. Apesar que ñ sei direito como fazer link pro rodapé...